src/storage/v2/storage.cpp

```diff
@@ -117,7 +117,7 @@ StorageMode Storage::Accessor::GetCreationStorageMode() const { return creation_
 
 std::optional<uint64_t> Storage::Accessor::GetTransactionId() const {
   if (is_transaction_active_) {
-    return transaction_.transaction_id.load(std::memory_order_acquire);
+    return transaction_.transaction_id;
```

Comment:
why did we remove load again?

Reply:
Because this is not used in atomic way. transaction.id is set once and isn't changed or removed any other time